content/docs/guides/branching-neon-cli.md

```diff
@@ -50,19 +50,56 @@ connection_uris
 The Neon CLI provides a `neonctl connection-string` command you can use to extract a connection uri programmatically. See [Neon CLI commands — connection-string](https://neon.tech/docs/reference/cli-connection-string).
 </Admonition>
 
+## Create a branch from a non-primary parent
```

Comment:
this stuff doesn't look like it is related to the topic of pr, is it an intentional change?

Reply:
You're right. It was some missing content that we added while in these files.